articles/quickstart/native/android-vnext/01-user-metadata.md

```diff
@@ -0,0 +1,127 @@
+---
+title: User Metadata
+description: This quickstart demonstrates how to make API calls to the Auth0 Management API to read and update user metadata.
+seo_alias: android
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - android
+  - api
+github:
+  path: 00-Login-Kt
+sample_download_required_data:
+  - client
+  - api
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD002 MD041 -->
+
+::: note
+Before you continue with this tutorial, make sure that you have completed the [Login chapter](/quickstart/native/android-vnext/00-login)
+:::
+
+As an example of using the SDK to call APIs that may require custom scopes and audiences, we will demonstrate these settings by reading and updating user profile information.
+
+## Read User Metadata
+
+In [the previous chapter](/quickstart/native/android-vnext/00-login#show-user-profile-information), you saw how to access [basic user profile information](https://auth0.com/docs/users/user-profiles) using the `/userinfo` endpoint. This profile information may not include everything about a user that you might need. For example, in order to access the user's metadata, you must make an API call to the Management API using the user ID and the access token. With the Android SDK, this is done using the `UsersAPIClient` object.
+
+To be able to correctly read and update the user metadata, ensure the authorization server allows you to read and edit the current user profile by specifying the `profile email read:current_user update:current_user_metadata` scopes. In addition, specify the audience for the [Auth0 Management API](https://auth0.com/docs/api), which happens to include the User Info audience as well.
+
+Find the code in your project that initializes the `WebAuthProvider` class, and make the following changes if necessary:
+
+* amend the argument given to `withScope` to be `"openid profile email offline_access read:current_user update:current_user_metadata"`
+* add a call to `withAudience` and pass the audience for the Auth0 Management API: `"https://${account.namespace}/api/v2"`.
```

Comment:
Typo. May be worth pointing out this specifically as could be a hard to track cause of issues.

```suggestion
* add a call to `withAudience` and pass the audience for the Auth0 Management API: `"https://${account.namespace}/api/v2/"`.
```